src/main/java/calculator/NumberCalculator.java

```diff
@@ -0,0 +1,14 @@
+package calculator;
+
+import java.util.List;
+
+public class NumberCalculator {
+
+	public static final int DEFAULT_VALUE = 0;
+
+	public int calculate(List<Integer> numbers) {
+		return numbers
+			.stream()
+			.reduce(DEFAULT_VALUE, Integer::sum);
+	}
```

Comment:
코드는 작성한 뒤, 항상  IDE의 자동 정렬 기능을 이용해서 정렬해주세요! (Mac 기준으로`⌥ + ⌘  + L`, Window는 확실치는 않지만 `Ctrl + Alt + L`) 
의식적으로 진행하시기 힘들다면 [파일 저장시 자동으로 정렬](https://hianna.tistory.com/791)을 수행하도록 설정하는 방법도 있어요. 😃

제가 자동정렬을 진행하면 아래와 같이 나오는데요. 겉보기에는 큰 차이가 없어보이지만, 들여쓰기를 Tab 대신에 space 4칸한다는 차이점이 있어요. 이런 내용들도 미션 소개 페이지에서 소개하고있는 자바 코드 컨벤션 문서에 정리되어있으니 하나를 참고해서 반영해주시면 좋을 것 같습니다! 😃 (아래에 자바 코딩 규칙 문서를 참고해주시면 될 것 같아요. 😉)

```suggestion
	public int calculate(List<Integer> numbers) {
            return numbers
                .stream()
                .reduce(DEFAULT_VALUE, Integer::sum);
        }
```

[자바 코딩 규칙 - 들여쓰기](https://myeonguni.tistory.com/1596)를 참고해보시면 좋을 것 같습니다. 😃